dt-users/template-new-user.php

```diff
@@ -3,6 +3,27 @@
  * Name: User Management
 */
 
+// Ensure Add New Users' admin setting has been enabled
+if ( current_user_can( 'manage_dt' ) && ! current_user_can( 'create_users' ) && ! DT_User_Management::non_admins_can_make_users() ) {
```

Comment:
@kodinkat This is not the same setting that is change in the network section

Reply:
Reverted back to `create_users` capability check only; which can be tested by toggling Add New Users network setting.

Reply:
@kodinkat Let's still replace `DT_User_Management::non_admins_can_make_users()` with a check for the network setting